ilastik/applets/objectExtraction/objectExtractionGui.py

```diff
@@ -229,6 +229,25 @@ def recursiveCheckChildren(twitem, val, exclude_location=False):
                     if getattr(child, "group_name", ""):
                         if child.group_name != "Location":
                             child.setCheckState(0, val)
+                
+    @staticmethod 
+    def updateParentState(item):
```

Comment:
```suggestion
    def updateParentCheckState(item):
```

"State" is a very general term, this is a specific method